openedx/core/djangoapps/site_configuration/helpers.py

```diff
@@ -29,6 +29,17 @@ def get_current_site_configuration():
         return None
 
 
+def get_current_site_configuration_values(default=None):
+    """
+    Returns `SiteConfiguration.site_values` for current site.
+    """
+    site_configuration = get_current_site_configuration()
+    if site_configuration:
+        return site_configuration.site_values
+    else:
+        return default if default else {}
```

Comment:
@shimulch, is there are a reason why `default` should default to `None` instead of `{}` which will simplify the code in this function to something like

```suggestion
    site_configuration = get_current_site_configuration()
    return site_configuration.site_values if site_configuration else default
```

Reply:
Thanks for providing the simplification. I might have overlooked the implementation here somehow and failed to come up with this simpler code.